dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -325,12 +327,30 @@ internal constructor(private val handler: LogHandler) {
     /**
      * Add a custom attribute to all future logs sent by this logger.
      * @param key the key for this attribute
-     * @param value the (nullable) Date value of this attribute
+     * @param value the (nullable) [Date] value of this attribute
      */
     fun addAttribute(key: String, value: Date?) {
         attributes[key] = value
     }
 
+    /**
+     * Add a custom attribute to all future logs sent by this logger.
+     * @param key the key for this attribute
+     * @param value the (nullable) [JsonObject] value of this attribute
```

Comment:
Don't you want to add additional note on nested objects limitation? 

When 10 nested levels are exceeded, the log will be dropped. I think this is important to mention it either in this inline documentation or to print a warning for the library user when parsing the log. Now we do none of those.